### ISSUE TITLE
Make notification area smaller

### ISSUE THREAD

**Issue by CTimmerman** [safing/portmaster#684]
**What would you like to add or change?**:

Make the notification area smaller when Portmaster is next to another window. Side-by-side makes Portmaster GUI cramped.
The scrollbars don't work properly to make everything visible either, so this could also be a bug report.

![image](https://user-images.githubusercontent.com/9662163/172352475-f444b4d4-a76a-4982-834e-299fd3fbe572.png)

**Why do you and others need this?**:

Right now the useful info is hidden in favor of a largely empty prompt and notification area, which can be relegated to a side icon with a notification badge.

Also, Attempting to submit this request from Portmaster (Advanced Interface) using my GitHub account fails with: "Invalid preset suggest-feature.md"

**Comment by davegson** [comment]
Hey there,

thanks for using the PM and for your input!

We definitely agree - this is a thing we have discussed as a team already. It's just a matter of priorities, since we have limited resources.

As a heads up, within the "Beta" phase - reaching it is our current focus - the UI/UX will get more love.

Also, the attached bug is really odd since it [that template is here](https://github.com/safing/portmaster/tree/develop/.github/ISSUE_TEMPLATE) and has been since forever. And I have used it before and it worked... 🤔
